site/docs/4.3/components/forms.md

```diff
@@ -176,7 +176,7 @@ Default checkboxes and radios are improved upon with the help of `.form-check`,
 
 Disabled checkboxes and radios are supported. The `disabled` attribute will apply a lighter color to help indicate the input's state.
 
-Checkboxes and radios use are built to support HTML-based form validation and provide concise, accessible labels. As such, our `<input>`s and `<label>`s are sibling elements as opposed to an `<input>` within a `<label>`. This is slightly more verbose as you must specify `id` and `for` attributes to relate the `<input>` and `<label>`.
+Checkboxes and radio buttons are used to build and support HTML-based form validation and provide concise, accessible labels. As such, our `<input>`s and `<label>`s are sibling elements as opposed to an `<input>` within a `<label>`. This is slightly more verbose as you must specify `id` and `for` attributes to relate the `<input>` and `<label>`.
```

Comment:
Even shorter: Checkboxes and radio buttons support HTML-based form validation ...

Reply:
Even better. Let´s go with the short version.